packages/mapviewer/src/modules/menu/components/help/MenuHelpSection.vue

```diff
@@ -26,7 +26,10 @@ const store = useStore()
 const hasGiveFeedbackButton = computed(() => store.getters.hasGiveFeedbackButton)
 const hasReportProblemButton = computed(() => store.getters.hasReportProblemButton)
 
-const currentLang = ref(store.state.i18n.lang)
+const currentLang = computed(() => store.state.i18n.lang)
+
+// this is needed to pass the selected value to the changelang function
+const selectedLang = ref(currentLang.value)
```

Comment:
why this change?

Reply:
because when trying to change the lang in the URL on mobile, it would not work, which caused some tests to fail.